docs/source/index.rst

```diff
@@ -1,21 +1,130 @@
+===========================
+Beamer Bridge documentation
+===========================
+
+Welcome! This is the official documentation for Beamer Bridge.
+
+Beamer is a bridging protocol lets you transfer your ERC20 tokens between supported L2 chains and 
+it also supports bridging to L1.
+
+Why Beamer?
+-----------
+
+* *Secure*: We don't use any liquidity pool or smart contract to deposit the liquidity. This effectively reduces to get hacked.
+* *Seamless*: Our `app <https://app.beamerbridge.com/>`_ is user-friendly and easy to use. 
+* *Fast*: Transfers are the fastest. You can check our speed comparison `here <https://twitter.com/BeamerBridge/status/1603043379331620865?s=20>`_.
+
+The Beamer Protocol
+-------------------
+
+Beamer is a protocol to enable users to move tokens from one chain to another. The user requests a transfer by
+providing tokens on the source chain. Liquidity providers then fill the request and directly send tokens to the user
+on the target chain. Check out :ref:`protocol` to learn more.
+
+Beamer Contracts
+----------------
+
+Beamer is using set of contracts to fulfill transfer requests. More basic ones used are 
+``RequestManager`` and ``FillManager``. In order to add more security Beamer uses L1->L2 and L2->L1 messaging through
+messenger contracts to use L1 resolutions. Check out sections below to find out more.
+
+1. :ref:`contracts-architecture`
+
+2. :ref:`contracts-parameters`
+
+3. :ref:`contracts-l1-resolution`
+
+4. :ref:`reference-contract-parameters`
+
+Running an Agent
+----------------
+
+Beamer is providing a software for liquidity providers called agent. This software is a docker image so it is really easy
+to use and setup. Check out the sections below if you want to more about the software.
+
+1. :ref:`running-agent-prerequisites`
+
+2. :ref:`running-agent-configuration`
+
+3. :ref:`running-agent-deployment-info`
+
+4. :ref:`running-agent-starting`
+
+5. :ref:`running-agent-stopping`
+
+6. :ref:`running-agent-troubleshooting`
+
+Development
+-----------
+
+Developers who want to contribute to the protocol can check sections below.
+
+1. :ref:`development-overview`
+
+2. :ref:`development-getting-started`
+
+3. :ref:`development-agent`
+
+4. :ref:`development-release`
+
+5. :ref:`development-branching`
+
+Reference documentation
+-----------------------
+
+In following sections, you can find detailed information about our software and contracts.
+
+1. :ref:`reference-commandline`
+
+2. :ref:`reference-configuration`
+
+3. :ref:`reference-contract-parameters`
+
+Current Deployment
+------------------
+
+This page consists of current contract deployment addresses. :ref:`deployment`
+
+Audit
+-----
+
+You can check contract audit reports in this page. :ref:`audit`
+
+FAQ
+---
+
+You can see the answers for questions in this page. :ref:`faq`
+
+Glossary
+--------
+
+This page has explanations to terms used in this docs. :ref:`glossary`
+
+
+Are you interested?
+-------------------
```

Comment:
I would avoid questions in our docs. Instead, something like `Community links` or similar would seem more appropriate. Could you please take a quick look at how other projects do it?